style.css

```diff
@@ -742,3 +742,239 @@ textarea {
+
+  .primary-menu {
+    color: #344563;
+    font-family: "Poppins", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  #hamburger {
+    display: none;
+  }
+
+  .modal {
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  .modal-content-body {
+    display: grid;
+    grid-template-columns: 60% 40%;
+  }
+
+  .modal-content-body .text {
+    padding-right: 30px;
+  }
+
+  .modal-btns .btn-project {
+    padding: 5px;
+  }
+
+  main.open-modal .menuToggle {
+    background: #c1c7d0;
+  }
+
+  .background h2 {
+    color: #091e42;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-size: 48px;
+    font-weight: bold;
+    line-height: 60px;
+    display: flex;
+    margin-left: 290px;
+    letter-spacing: 0.37px;
+    padding: 8px;
+  }
+
+  .intro {
+    color: #344563;
+    font-family: "Poppins", sans-serif;
+    font-weight: normal;
+    font-size: 20px;
+    display: flex;
+    line-height: 28px;
+    margin-left: 300px;
+    margin-right: 200px;
+  }
+
+  .card-img {
+    width: 544px;
+    height: 448px;
+    transition: transform;
+  }
+
+  .card-img:hover {
+    transform: scale(1.02);
+  }
+
+  .card {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: auto auto;
+    padding: 24px;
+  }
+
+  .social-icons {
+    margin-left: 300px;
+  }
+
+  .card-body {
+    display: flex;
+    flex-direction: column;
+    padding-left: 20px;
+  }
+
+  .invert {
+    order: -1;
+  }
+
+  form div {
+    display: flex;
+    justify-content: center;
+  }
+
+  /* NEW CODE FOR THE POST CONTAINER */
+
+  .post-section {
+    padding-left: 100px;
+    padding-right: 100px;
+  }
+
+  .post-container {
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    margin: 0 auto;
+  }
+
+  .post-container > div {
+    margin: 0 0 0 0;
+  }
+
+  .post-container div:last-child {
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .post-container .left {
+    flex: 1 1 50%;
+    padding: 0;
+  }
+
+  .image {
+    object-fit: contain;
+    height: 100%;
+  }
+
+  .post-container .right {
+    width: 50%;
+    padding: 10px 0 0 0;
+  }
+
+  .order-1 {
+    order: 1;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  .about {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: 1fr 1fr;
+    padding: 10%;
+  }
+
+  .about-desk h1 {
+    font-family: "Poppins", Arial, Helvetica, sans-serif;
+    line-height: 52px;
+    font-style: normal;
+    font-weight: bold;
+    color: #091e42;
+  }
+
+  .desk-about-section {
+    display: flex;
+    width: 100%;
+  }
+
+  .about-text p {
+    width: 90%;
+    padding: 10px;
+    margin-right: 0%;
+    padding-right: 0%;
+  }
+
+  .java {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .language {
+    width: 60%;
+    padding-top: 30px;
+  }
+
+  .language ul {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
- [ ] As mentioned by the previous reviewer, the language part of this section takes up 60% and the content is aligned to the right. But the width of the parent container of that part takes the whole 60%. With that, you make the underlines of both frameworks and skills to take the whole width. The language and downward arrow can be styled with display: flex and justify-content: space-between to make them stay apart with each move to the left and right edges. Kindly look at them very closely and fix them.

### Your Design
![your-about-page](https://user-images.githubusercontent.com/26597432/215277949-f7120abe-1058-42ce-a4b9-26d1d319d6ac.png)

### Figma Design
![temp-design](https://user-images.githubusercontent.com/26597432/215277957-006539f7-5aa3-4c3c-9610-01a901b0b6f2.png)